tests/test_dm.py

```diff
@@ -0,0 +1,34 @@
+from request import sogs_get, sogs_post
+from sogs import crypto
+
+from base64 import b64encode
```

Comment:
```suggestion
from sogs.utils import encode_base64
```

Reply:
The one from utils includes the trailing `.decode()` which is virtually always what you want.  `base64` just likes being obtuse in every way.